internal/env/interface.go

```diff
@@ -0,0 +1,34 @@
+package env
+
+type Loader struct {
+	original []iterItem
+	envdir   string
+}
+
+type Iterator struct {
+	ch    chan *iterItem
+	nextK string
+	nextV string
+}
+
+type iterItem struct {
+	key   string
+	value string
+}
+
+type Environment interface {
+	Clearenv()
+	Setenv(string, string)
+}
+type sysenv struct{}
+
+type Option interface {
+	Name() string
+	Value() interface{}
+}
+
+const LoadEnvdirKey = "LoadEnvdirKey"
```

Comment:
If all we want is the `LoadEnvdirKey` flag, then having the interface and the struct feels too much of generalization. What do you think?